src/main/java/io/seqera/tower/cli/responses/TaskView.java

```diff
@@ -0,0 +1,151 @@
+        this.times = times;
+        this.resources = resources;
+        this.usage = usage;
+    }
+
+    @Override
+    public Object getJSON() {
+        Map<String, Object> data = new HashMap<>();
+
+        data.put("general", general);
+
+        if (command != null) {
+            data.put("command", command);
+        }
+
+        if (environment != null) {
+            data.put("environment", environment);
+        }
+
+        if (!times.isEmpty()) {
+            data.put("times", times);
+        }
+
+        if (!resources.isEmpty()) {
+            data.put("resources", resources);
+        }
+
+        if (!usage.isEmpty()) {
+            data.put("usage", usage);
+        }
+
+        return data;
+    }
+
+    @Override
+    public void toString(PrintWriter out) {
+        out.println(ansi(String.format("%n%n    @|bold  %s|@", "General")));
+        TableList tableGeneral = new TableList(out, 2);
+        tableGeneral.setPrefix("    ");
+        tableGeneral.addRow("Task ID", general.get("taskId").toString());
+        tableGeneral.addRow("ID", general.get("id").toString());
+        tableGeneral.addRow("Work directory", general.get("workDir").toString());
+        tableGeneral.addRow("Status", general.get("status").toString());
+        tableGeneral.print();
+
+        if (command != null) {
+            out.println(ansi(String.format("%n     @|bold Command|@")));
+            out.println(ansi("    ---------"));
+            out.println(ansi(String.format("     %s", command.trim().replaceAll("\n", String.format("%n     ")))));
+        }
+
+        if (environment != null) {
+            out.println(ansi(String.format("%n     @|bold Environment|@")));
+            out.println(ansi("    -------------"));
+            out.println(ansi(String.format("     %s", environment.trim().replaceAll("\n", String.format("%n     ")))));
+        }
+
+        if (!times.isEmpty()) {
+            out.println(ansi(String.format("%n%n    @|bold  %s|@%n", "Execution time")));
+            TableList tableTime = new TableList(out, 3, "Label", "Value", "Description");
+            tableTime.setPrefix("    ");
+            tableTime.addRow("submit", FormatHelper.formatTime((OffsetDateTime) times.get("submit")), "Timestamp when the task has been submitted");
+            tableTime.addRow("start", FormatHelper.formatTime((OffsetDateTime) times.get("start")), "Timestamp when the task execution has started");
+            tableTime.addRow("complete", FormatHelper.formatTime((OffsetDateTime) times.get("complete")), "Timestamp when task execution has completed");
+            tableTime.addRow("duration", FormatHelper.formatDurationMillis((Long) times.get("duration")), "Time elapsed to complete since the submission i.e. including scheduling time");
+            tableTime.addRow("realtime", FormatHelper.formatDurationMillis((Long) times.get("realtime")), "Task execution time i.e. delta between completion and start timestamp i.e. compute wall-time\n");
```

Comment:
Remove the ending `\n` most likely this is what makes the windows binary tests fail.